src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
Both Arial and Georgia are proprietary fonts with many  [restrictions](https://www.microsoft.com/typography/faq/faq8.htm): "You may only redistribute the fonts in their original form (.exe or .sit.hqx) and with their original file name from your Web site or intranet site.
You must not supply the fonts, or any derivative fonts based on them, in any form that adds value to commercial products, such as CD-ROM or disk based multimedia programs, application software or utilities." At the end you have to pay...